.cirrus.yml

```diff
@@ -110,16 +116,20 @@ task:
       build_script:
         - flutter config --enable-linux-desktop
         - ./script/incremental_build.sh build-examples --linux
-      test_script:
```

Comment:
Why is this removed? Separating these is helpful in looking at results.

Reply:
Nice catch, merge conflict editing problem. Updating!